ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -318,56 +350,458 @@ Loader {
+                case Constants.messageContentType.stickerType:
+                    return StatusMessage.ContentType.Sticker;
+                case Constants.messageContentType.emojiType:
+                    return StatusMessage.ContentType.Emoji;
+                case Constants.messageContentType.transactionType:
+                    return StatusMessage.ContentType.Transaction;
+                case Constants.messageContentType.imageType:
+                    return StatusMessage.ContentType.Image;
+                case Constants.messageContentType.audioType:
+                    return StatusMessage.ContentType.Audio;
+                case Constants.messageContentType.communityInviteType:
+                    return StatusMessage.ContentType.Invitation;
+                case Constants.messageContentType.fetchMoreMessagesButton:
+                case Constants.messageContentType.chatIdentifier:
+                case Constants.messageContentType.unknownContentType:
+                case Constants.messageContentType.statusType:
+                case Constants.messageContentType.systemMessagePrivateGroupType:
+                case Constants.messageContentType.gapType:
+                case Constants.messageContentType.editType:
+                default:
+                    return StatusMessage.ContentType.Unknown;
+                }
+            }
+
+            readonly property int contentType: convertContentType(root.messageContentType)
+            readonly property bool isReply: root.responseTo !== ""
+            readonly property var replyMessage: root.messageStore && isReply ? root.messageStore.getMessageByIdAsJson(root.responseTo) : null
+            readonly property string replySenderId: replyMessage ? replyMessage.senderId : ""
+
+            function editCompletedHandler(newMessageText) {
+                const message = root.rootStore.plainText(StatusQUtils.Emoji.deparse(newMessageText))
+                if (message.length <= 0)
+                    return;
+
+                const interpretedMessage = root.messageStore.interpretMessage(message)
+                root.messageStore.setEditModeOff(root.messageId)
+                root.messageStore.editMessage(root.messageId, interpretedMessage)
+            }
+
+            audioMessageInfoText: qsTr("Audio Message")
+            cancelButtonText: qsTr("Cancel")
+            saveButtonText: qsTr("Save")
+            loadingImageText: qsTr("Loading image...")
+            errorLoadingImageText: qsTr("Error loading the image")
+            resendText: qsTr("Resend")
+            pinnedMsgInfoText: qsTr("Pinned by")
+            reactionIcons: [
+                Style.svg("emojiReactions/heart"),
+                Style.svg("emojiReactions/thumbsUp"),
+                Style.svg("emojiReactions/thumbsDown"),
+                Style.svg("emojiReactions/laughing"),
+                Style.svg("emojiReactions/sad"),
+                Style.svg("emojiReactions/angry"),
+            ]
```

Comment:
That's strange we don't use the same emojis for reactions as these we send in the chat message (twemoji). I couldn't find designs for reactions, do you have the link maybe?

Reply:
seems it matches the design :eyes: thanks for finding the link